hhs_oauth_server/request_logging.py

```diff
@@ -399,7 +399,7 @@ def to_dict(self):
                 ).hexdigest()
                 self.log_msg["access_token_scopes"] = " ".join([s for s in at.scopes])
                 self._log_msg_update_from_object(
-                    at.application, "access_token_id", "id"
+                    at, "id", "id"
```

Comment:
```suggestion
                    at, "access_token_id", "id"
```
I think the old name was okay, we just needed to update how it was populated.